src/components/shared/Button/SharedButton.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import { defaultButtonStyle } from './Button.styles';
+
+function SharedButton({ label, style = {}, ...rest }) {
+  return (
+    <Button sx={{ ...defaultButtonStyle, ...style }} {...rest}>
+      {label}
```

Comment:
You should create styled component for your button, and then pass the style object to allow style override